packages/solid/components/Column/Column.tsx

```diff
@@ -15,42 +15,31 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { type Component } from 'solid-js';
+import { createMemo, type Component } from 'solid-js';
 import { View } from '@lightningtv/solid';
 import type { KeyHandler } from '@lightningtv/solid/primitives';
 import { handleNavigation, onGridFocus } from '../../utils/handleNavigation.js';
-import { withScrolling, type ScrollableElement } from '../../utils/withScrolling.js';
+import { withScrolling } from '../../utils/withScrolling.js';
 import { chainFunctions } from '../../utils/chainFunctions.js';
 import styles from './Column.styles.js';
-import type { ColumnProps, ScrollableComponent } from './Column.types.js';
```

Comment:
if we dont need ScrollableComponent anymore we should remove it from Column.types as well